docs/feature-gates/conditions.mdx

```diff
@@ -6,35 +6,41 @@ slug: /feature-gates/conditions
 
 ## Introduction
 
-Statsig feature gates contain a list of **rules**, which are evaluated in order from top to bottom.
-As soon as a user passes the **condition(s)** in a given rule, subsequent rules will be ignored.
-The user is then evaluated against the percentage rollout of that rule, passing if they meet the threshold and failing otherwise.
+Statsig feature gates contain a list of rules that are evaluated in order from top to bottom. The page describes in more detail how these rules are evaluated and lists all currently supported conditions.
 
-:::warn
-WARNING - order matters for rules!  If a user meets a condition at the top, evaluation stops (regardless of if they pass or fail the rollout percentage).
-Another way to think of it, using boolean logic: rules are combined with an OR (short circtuiting when one is true), while conditions are combined with AND.
-:::
+### Rule Evaluation
 
-If the user passes condition 1 and 2, condition 3 is never evaluated.  The rollout percentage of rule 1 would then be evaluated against the userID and salt of the gate.
+The rules that you create are evaluated in the order you create them. For each rule, the **criteria** or **conditions** determine which users _qualify_ for the Pass/Fail treatments. The Pass filter percentage further determines the percentage of _qualifying_ users that will be exposed to the new feature. 
```

Comment:
Thats not actually true - you can rearrange the order of the rules after you create them.  Its more that they are evaluated in order from top to bottom